po/rufus.pot

```diff
@@ -86,8 +86,9 @@ msgctxt "LightDownloadDescription"
 msgid "Includes basic features of Endless OS and the option to download more apps and content later. This version is best for computers with reliable internet access."
 msgstr ""
 
+#. This means "Download [the] Basic [edition of Endless OS]". Use the same translation as for the "BASIC" string.
 msgctxt "DownloadLightButton"
-msgid "Download EOS Basic"
+msgid "Download Basic"
```

Comment:
I noticed that many translations of DownloadLightButton and DownloadFullButton don't match up. They should be the same in all but one word, which should be the same as the BASIC/FULL translation. But some, eg. Arabic, Hindi and Bangla, include "EOS Full" verbatim for one but only "EOS" for the other.

So I plan to update the obvious ones (ie languages I vaguely understand) to remove "EOS", but ask the respective translators to update the others.